pypit/arsave.py

```diff
@@ -462,6 +470,51 @@ def save_1d_spectra_fits(slf, clobber=True):
     #with io.open(sensfunc_name, 'w', encoding='utf-8') as f:
     #    f.write(unicode(json.dumps(slf._sensfunc, sort_keys=True, indent=4, separators=(',', ': '))))
 
+def save_obj_info(slf, fitsdict, clobber=True):
+    # Lists for a Table
+    names, boxsize, opt_fwhm, s2n = [], [], [], []
+    # Loop on detectors
+    for kk in range(settings.spect['mosaic']['ndet']):
+        det = kk+1
+        dnum = settings.get_dnum(det)
+        # Loop on spectra
+        for specobj in slf._specobjs[det-1]:
+            # Append
+            names.append(specobj.idx)
+            # Boxcar width
+            if 'size' in specobj.boxcar.keys():
+                slit_pix = specobj.boxcar['size']
+                # Convert to arcsec
+                binspatial, binspectral = settings.parse_binning(fitsdict['binning'][specobj.scidx])
+                boxsize.append(slit_pix*binspatial*settings.spect[dnum]['platescale'])
+            else:
+                boxsize.append(0.)
+            # Optimal profile (FWHM)
+            if 'fwhm' in specobj.optimal.keys():
+                binspatial, binspectral = settings.parse_binning(fitsdict['binning'][specobj.scidx])
+                opt_fwhm.append(specobj.optimal['fwhm']*binspatial*settings.spect[dnum]['platescale'])
+            else:
+                opt_fwhm.append(0.)
+            # S2N -- default to boxcar
+            sext = (specobj.boxcar if (len(specobj.boxcar) > 0) else specobj.optimal)
+            ivar = (sext['var'] == 0) / (sext['var'] + (sext['var'] == 0))
```

Comment:
I think I might be confused here, but I'll make a note just incase this is a bug... Should this be:
`ivar = (sext['var'] != 0) / (sext['var'] + (sext['var'] == 0))`

In the current version of the code, if sext['var'] is 0, then the numerator will be 1 and the denominator will be 1 (i.e. ivar=1). Then, if sext['var'] is not 0, then the numerator will be 0 and ivar will be 0. So isn't the current version of the code just giving 1 and 0 for ivar?

Perhaps I'm missing something?